tests/tests/plugin/unit/test_trainable_tagger.py

```diff
@@ -68,7 +69,7 @@ def test_trainable_tagger():
         ),
         model,
     )
-    assert tagger1.data == TRAIN_RESPONSE.dict()
+    assert tagger1.data == {to_camel(k): v for (k, v) in TRAIN_RESPONSE.dict().items()}
```

Comment:
@EniasCailliau I don't think my change here should be necessary given my understanding of `CamelModel`... so I wanted to flag you to get your thoughts -- am I doing something wrong in the construction of TRAIN_RESPONSE?

Reply:
We need to enable `use_alias` while calling `dict()`: 

```
TRAIN_RESPONSE.dict(use_alias=True)
```

Reply:
Thanks!